keyboards/planck/keymaps/muppetjones/keymap.c

```diff
@@ -0,0 +1,441 @@
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT_planck_wrapper(
+    __ADJUST_L1________________________________,   AU_ON,  AU_OFF, __ADJUST_R1________________________________,
+    __ADJUST_L2________________________________,  MUV_DE,  MUV_IN, XXXXXXX, CLMK_DH, QWERTY,   PLOVER, XXXXXXX,
+    __ADJUST_L3________________________________,   MI_ON,  MI_OFF, __ADJUST_R3________________________________,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,  MU_MOD,   MU_ON,  MU_OFF, __BLANK_NOOP_______________________________
+)
+
+/*
+ * ,-----------------------------------------------------------------------------------.
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+[_ADJUST] = LAYOUT_planck_grid(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+)
+
+ */
+    // clang-format on
+};
+
+#ifdef AUDIO_ENABLE
+float plover_song[][2]    = SONG(PLOVER_SOUND);
+float plover_gb_song[][2] = SONG(PLOVER_GOODBYE_SOUND);
+#endif
+
+/* Setup layer lighting
+#define HSV_WHITE 0, 0, 255
+#define HSV_RED 0, 255, 255
+#define HSV_CORAL 11, 176, 255
+#define HSV_ORANGE 28, 255, 255
+#define HSV_GOLDENROD 30, 218, 218
+#define HSV_GOLD 36, 255, 255
+#define HSV_YELLOW 43, 255, 255
+#define HSV_CHARTREUSE 64, 255, 255
+#define HSV_GREEN 85, 255, 255
+#define HSV_SPRINGGREEN 106, 255, 255
+#define HSV_TURQUOISE 123, 90, 112
+#define HSV_TEAL 128, 255, 128
+#define HSV_CYAN 128, 255, 255
+#define HSV_AZURE 132, 102, 255
+#define HSV_BLUE 170, 255, 255
+#define HSV_PURPLE 191, 255, 255
+#define HSV_MAGENTA 213, 255, 255
+#define HSV_PINK 234, 128, 255
+#define HSV_BLACK 0, 0, 0
+#define HSV_OFF HSV_BLACK
+ *
+ */
```

Comment:
```suggestion
```

Reply:
Leaving this was intentional, and as comment lines aren't included at compile time, it doesn't hurt to leave them in a personal keymap.

Reply:
Why duplicate what the docs already provide?

Reply:
I copied from there. I was experimenting, and wanted a more direct reference so I didn't have to look it up each time. I figured that since it was a comment, it wasn't hurting anything.

This experiment culminated in a dynamic layer lighting that I haven't moved into my user space yet.